kvmagent/kvmagent/plugins/tensorfusion/service.py

```
@@ -42,11 +42,38 @@ def initialize(self):
+                     '(%d orphans cleaned)' % (len(alive), len(running) - len(alive)))
         self._monitor.start()
 
+    @staticmethod
+    def _is_vm_running(vm_uuid):
+        """Check if a VM is still running in libvirt."""
+        try:
+            from kvmagent.plugins.vm_plugin import LibvirtAutoReconnect
+            with LibvirtAutoReconnect() as conn:
+                dom = conn.lookupByName(vm_uuid)
+                return dom.isActive() == 1
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)